server/graphql/purchase_order_line/src/purchase_order_line_queries.rs

```diff
@@ -14,11 +14,7 @@ use repository::{
 #[graphql(rename_items = "camelCase")]
 pub enum PurchaseOrderLineSortFieldInput {
     ItemName,
-    NumberOfPacks,
     LineNumber,
-    RequestedQuantity,
```

Comment:
We probably want some of these back but though better to remove for now until we're sure...